samples/custom-service/pom.xml

```
@@ -0,0 +1,84 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+						http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+ 
+	<groupId>com.dynatrace.oneagent.sdk.samples.customservice</groupId>
+	<artifactId>custom-service-sample</artifactId>
+	<version>1.8.0</version>
+	<packaging>jar</packaging>
+ 
+	<dependencies>
+	<dependency>
+		<groupId>com.dynatrace.oneagent.sdk.java</groupId>
+		<artifactId>oneagent-sdk</artifactId>
+		<version>1.8.0</version>
+		<scope>compile</scope>
+	</dependency>
+	</dependencies>
+
+	<url>https://github.com/Dynatrace/OneAgent-SDK-Java</url>
+	<name>Dynatrace OneAgent SDK Java custom service sample</name>
+	<organization>
+		<name>Dynatrace</name>
+		<url>http://www.dynatrace.com</url>
+	</organization>
+	
+	<properties>
+	<maven.compiler.source>1.6</maven.compiler.source>
+	<maven.compiler.target>1.6</maven.compiler.target>
+	<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+	<!-- uncomment and replace url/tenant/token in case of manual agent injection -->
+	<!-- 
+	<agent.lib>path_to_installed_one_agent\agent\lib64\oneagentjava.dll</agent.lib>
+	<agent.server>https://url.to.tenant.com:443</agent.server>
```

```suggestion
	<agent.server>https://url.to.tenant.com</agent.server>
```

Isn't 443 the default for HTTPS?